src/test/java/api/requests/RequestsAPIMoveTests.java

```diff
@@ -308,7 +276,7 @@ public void canMoveAHoldShelfRequestToAnEmptyQueue()
 
     // make requests for smallAngryPlanet
     IndividualResource requestByJessica = requestsFixture.placeHoldShelfRequest(
-      smallAngryPlanet, jessica, new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC));
```

Comment:
Is the relevant part of this test that it is a `hold` request type, or that the fulfilment preference is `to the hold shelf`?

Reply:
Just exhaustive testing. Might need better assertions. Any help on this would be appreciated.

Reply:
Sorry, this comment is likely out of place. I couldn't find a way to comment on an unchanged line. I really wanted to comment on the same of the test.

And it may well be that I misread where in the code this change was made. I think I'll resolve this.